From 38bcf37e46adc8150f21eb571730d9a15c721048 Mon Sep 17 00:00:00 2001
From: Pardha Kesapragada <pkesapragada@habana.ai>
Date: Tue, 16 Mar 2021 18:56:43 +0200
Subject: [PATCH] [SW-0] gaudi2_arm: increase stack size

Inline with the rest of the platforms,
increase the stack size for preboot for gaudi2
to 4KB.

Change-Id: Ib4b77cc3f0d7775227923a133c133270bab80144
Signed-off-by: Pardha Kesapragada <pkesapragada@habana.ai>
---

diff --git a/gaudi2_arm/preboot.lds.S b/gaudi2_arm/preboot.lds.S
index a610a7c..db72179 100644
--- a/gaudi2_arm/preboot.lds.S
+++ b/gaudi2_arm/preboot.lds.S
@@ -28,7 +28,7 @@
 	.stack : {
 		. = ALIGN(8);
 		__stack = . ;
-		. = . + 512 ;
+		. = . + 4096 ;
 		__stack_top = . ;
 	} > .sram
 
